apps/frontend/src/app/5_pages/RewardsPage/components/Staking/Staking.tsx

```diff
@@ -1,7 +1,104 @@
-import React, { FC } from 'react';
+import React, { FC, useMemo } from 'react';
 
-export const Staking: FC = () => (
-  <>
-    <div>TBD - Staking rewards table</div>
-  </>
-);
+import { formatUnits } from 'ethers/lib/utils';
+import { t } from 'i18next';
+
+import { SupportedTokens } from '@sovryn/contracts';
+import { Paragraph, Table } from '@sovryn/ui';
+
+import { AmountRenderer } from '../../../../2_molecules/AmountRenderer/AmountRenderer';
+import { BTC_RENDER_PRECISION } from '../../../../../constants/currencies';
+import { getTokenDisplayName } from '../../../../../constants/tokens';
+import { useAccount } from '../../../../../hooks/useAccount';
+import { translations } from '../../../../../locales/i18n';
+import { useGetFeesEarned } from '../../hooks/useGetFeesEarned';
+import { useGetLiquidSovClaimAmount } from '../../hooks/useGetLiquidSovClaimAmount';
+import { columns } from './Staking.constants';
+import { getStakingRevenueType } from './Staking.utils';
+import { WithdrawFee } from './components/WithdrawFee/WithdrawFee';
+import { WithdrawLiquidFee } from './components/WithdrawLiquidFee/WithdrawLiquidFee';
+
+export const Staking: FC = () => {
```

Comment:
Apparently, I have a really small amount of staking rewards available but `Withdraw` button is disabled.

<img width="1036" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/22981e01-db23-4f44-a2c6-078af4c04178">